.idea/compiler.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hi @eeijlar,

Thought you might want to remove these couple `compiler/vcs.xml` files from your first commit; the `.gitignore` was introduced afterwards, it doesn't retroactively apply. :)

In case those little details help moving forward,

Reply:
I have removed and update the pull request. I am stunned that anyone is even looking at this.

Thanks for the comments @mkgl